packages/alokai-cli/README.md

```diff
@@ -0,0 +1,41 @@
+Alokai CLI
+
+## Usage 
+
+### Adding new packages
+
+To add a new package to the Alokai CLI, you can run the following command:
+
+```bash
+pnpm add alokai-[your-package-name]
+```
+
+In order to use the package, you need to import it in the `index.ts` file:
+
+```typescript
+import alokaiCli[YourPackageName] from 'alokai-[your-package-name]';
+```
+
+Then, you can use the package in the `index.ts` file:
+
+```typescript
+alokaiCli[YourPackageName].forEach((command) => {
+    program.addCommand(command);
+  });
+```
+
+This will add the command to the core CLI application and will be available to use once the project is published.
```

Comment:
Imo it's better to create an abstraction over the commands' registration. We could have a following API

```ts
import integrationCommands from "alokai-integrations";

const program = createAlokaiCli({
  plugins: [
    { name: "integrations", commands: integrationCommands },
    { name: "modules", commands: modulesCommands },
    //...
  ]
});
```
or
```ts
import integrationCommands from "alokai-integrations";

const program = createAlokaiCli({
  plugins: {
    integrations: integrationsCommands,
    modules: modulesCommands,
    //...
  }
});
```